### ISSUE TITLE
Missing .test.js for the following components

### ISSUE THREAD

**Issue by DianeLiu2019** [cds-snc/report-a-cybercrime#2104]
##Summary

Missing .test.js for the following components: DatePicker, FormErrorMessage, FormHelperText, FormLabel, RedirectRoute(route)

**Comment by DianeLiu2019** [comment]
this issue is divided into 5 issues
